sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/BaseArgumentListSyntaxExtensionsTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2020 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Linq;
+using FluentAssertions;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarAnalyzer.Helpers;
+
+namespace SonarAnalyzer.UnitTest.Helpers
+{
+    [TestClass]
+    public class BaseArgumentListSyntaxExtensionsTests
+    {
+        [TestMethod]
+        public void GivenEmptyList_GetArgumentByName_ReturnsNull() =>
+            CreateNamedArgumentList().GetArgumentByName("argument").Should().BeNull();
+
+        [TestMethod]
+        public void GivenListWithAnotherNamedArgument_GetArgumentByName_ReturnsNull() =>
+            CreateNamedArgumentList("p1").GetArgumentByName("p2").Should().Be(null);
```

Comment:
For consistency, can we also have `Should().BeNull();` for `GivenListWithAnotherNamedArgument_GetArgumentByName_ReturnsNull` as it is done for `GivenEmptyList_GetArgumentByName_ReturnsNull`